backend/tests/prisonerPersonal.test.ts

```diff
@@ -2066,6 +2088,12 @@ describe('prisoner personal', () => {
       prisonApi.getPersonalCareNeeds.mockRejectedValue(new Error('Network error'))
       // @ts-expect-error ts-migrate(2339) FIXME: Property 'getReasonableAdjustments' does not exist... Remove this comment to see the full error message
       prisonApi.getReasonableAdjustments.mockRejectedValue(new Error('Network error'))
+      // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+      // @ts-ignore
+      prisonApi.userCaseLoads = jest.fn()
```

Comment:
Not sure why all these caseloads are needed? I thought we had agreed to use the prisoner caseload?

This applies to the prisonerProfile tests as well.

Reply:
Yes no need for caseloads now, removed!